### PR TITLE
mmuconfig to set threshold for profiles

### PR DESCRIPTION
**Pull request by alpeshspatel**
#### What I did
Extended the ***mmuconfig*** utility with a "-s / --staticth" keyword to configure the threshold configured via pg_profile_lookup.ini
The utility had ability to configure "-a/alpha" for dynamic threshold

#### How I did it
Added a keyword to set the threshold for static profiles

#### How to verify it
Configure via:
mmuconfig -p pg_lossless_400000_5m_profile -s 909090

dump the config via:
```
sonic# redis-dump -d 4 -y -k "BUFFER_PROFILE|pg_lossless_400000_5m_profile"
{
  "BUFFER_PROFILE|pg_lossless_400000_5m_profile": {
    "expireat": 1680565001.8800943,
    "ttl": -0.001,
    "type": "hash",
    "value": {
      "pool": "ingress_lossless_pool",
      "size": "0",
      "static_th": "909090",
      "xoff": "999000",
      "xon": "0"
    }
  }
```

### CODE REVIEW

**Comment by alpeshspatel** [comment]
@abdosi who to assign as reviewers for this repo?

File: scripts/mmuconfig [hunk 1]
```diff
@@ -124,14 +119,21 @@ class MmuConfig(object):
...
                 sys.exit("Invalid alpha value: 2^(%s)" % (value))
 
-            buf_profs = self.db.get_table(BUFFER_PROFILE_TABLE_NAME)
             if profile in buf_profs and DYNAMIC_THRESHOLD not in buf_profs[profile]:
                 sys.exit("%s not using dynamic thresholding" % (profile))
+        elif field == STATIC_THRESHOLD:
+            if v < 0:
+                sys.exit("Invalid static threshold value: (%s)" % (value))
+
+            buf_profs = self.db.get_table(BUFFER_PROFILE_TABLE_NAME)
```
Comment (rbpittman) [inline scripts/mmuconfig:134]: Duplicated line from above.

**Review by rbpittman — APPROVED** [review]

**Review by vmittal-msft — DISMISSED** [review]

**Comment by alpeshspatel** [comment]
@abdosi can you kindly review and do the needful?

**Review by abdosi — APPROVED** [review]

**Comment by alpeshspatel** [comment]
@abdosi we need in 202205 image as well. Can you add that label to merge in that branch as well?

**Pull request merged**